circuits/cpp/barretenberg/cpp/src/barretenberg/honk/flavor/goblin_ultra.hpp

```diff
@@ -29,15 +28,16 @@ namespace proof_system::honk::flavor {
 class GoblinUltra {
   public:
     using CircuitBuilder = UltraCircuitBuilder;
-    using PCSParams = pcs::kzg::Params;
-    using PCS = pcs::kzg::KZG<PCSParams>;
-    using Curve = PCSParams::Curve;
+    using Curve = curve::BN254;
+    using PCS = pcs::kzg::KZG<Curve>;
     using GroupElement = Curve::Element;
     using Commitment = Curve::AffineElement;
     using CommitmentHandle = Curve::AffineElement;
     using FF = Curve::ScalarField;
     using Polynomial = barretenberg::Polynomial<FF>;
     using PolynomialHandle = std::span<FF>;
+    using CommitmentKey = pcs::CommitmentKey<Curve>;
```

Comment:
Decided to change these to `CommitmentKey` and `VerifierCommitmentKey`. We already have a `VerificationKey` so that language is ambiguous even with the PCS prefix.  The verifier is actually using this object to compute commitments (in the full sense in IPA but also in Shplonk) so I think the name is appropriate and more descriptive. What do you think?

Reply:
sure, sounds reasonable!